src/main/python/RLBotFramework/agents/bot_manager.py

```diff
@@ -236,6 +236,35 @@ def prepare_for_run(self):
         pass
 
 
+class BotManagerFlatbuffer(BotManager):
```

Comment:
I feel like we should separate all the different bot managers into separate files.

There are now 4 of them

Reply:
There will be fewer soon, but I'll split it out anyway